### DISCUSSION TITLE
Not compiling? src dir

### DISCUSSION THREAD

**Discussion by fede843** [Dilbert66/esphome-dsckeybus#87] [General]
Hi, after updating dev branch today I was not able to compile the code. The reason was the file moving to src dir

**Comment by fede843** [comment]
Had to change 

```
  includes:
    - dscKeybusInterface/
```

by 

```
  includes:
    - src/
```

**Comment by fede843** [comment]
not sure if this is a general small refactor issue, or a miss configuration somewhere else in my local branch.

**Comment by Dilbert66** [comment]
That was my fault. I was trying something out with platformio.  I've renamed the "src" directory back to the original "dsckeybusinterface" to avoid confusion.

**Reply by fede843** [reply]
All good then!
